flowkit_ui_backend/util/config.py

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+# If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from functools import lru_cache
+from typing import Optional, Annotated
+
+from fastapi import Depends
+from pydantic import BaseSettings, Field
+from pydantic.networks import AnyHttpUrl
+from pydantic.types import SecretStr
+
+from flowkit_ui_backend.util.logging import LogLevel
+
+
+class Settings(BaseSettings):
+    db_port_container: int
+    db_pw: SecretStr
+    db_name: str = Field(
+        "flowkit_ui_backend", const=True
+    )  # This should really be configurable but is hardcoded in the schema at present
```

Comment:
Couldn't see at a glance where this is used now, as this param has been taken out of the hard-coded queries. Does it get checked at pool initialisation?